src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -173,7 +183,7 @@ public bool CheckBoxesEnabled
             _selectedCount = 0;
 
             // triggers the package list loader
-            LoadItems(selectedPackageItem, token);
+            LoadItems(selectedPackageItem, token, applyUIFilterUpdatesAvailable: tabToRender == ItemFilter.UpdatesAvailable);
```

Comment:
Mostly ignore for now - I was toying with forcing Updates to be treated as Installed loading (with later UI filter)